src/EPPlus/Drawing/Chart/ExcelChart.cs

```diff
@@ -1118,5 +1115,67 @@ void IStyleMandatoryProperties.SetMandatoryProperties()
         ZipPackagePart IPictureRelationDocument.RelatedPart => Part;
 
         Uri IPictureRelationDocument.RelatedUri => UriChart;
+
+
+
+        ExcelDrawings _chartDrawings = null;
+
+        /// <summary>
+        /// Collection of drawing-objects like shapes, images and charts
+        /// </summary>
+        public ExcelDrawings Drawings
```

Comment:
Create a new class with only the add methods that are possible to add .(Shape and Pictures)